about.html

```diff
@@ -7,62 +7,71 @@
+          <a href="#">Contact</a>
+        </nav>
+        <img src="img/jumbo-about.png">
+      </header>
+      <main>
+        <section class="top">
+        <h2>About</h2>
+        <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
+        </section>
+
+        <section class="middle">
+          <div class="mL">
+            <img src="img/about-plan.png" alt="strategy">
     
-    Collaboration
+            <h2>Strategy</h2>
+            <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+            <button>Learn More</button>
+          </div>
+          <div class="mR">
+            <img src="img/about-working.png" alt="strategy">
     
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-    Let's Work Together
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-        
+            <h2>How We Work</h2>
+            <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+            <button>Learn More</button>
+          </div>
+        </section>
+
+        <section class="middle">
+          <div class="mL"> 
+            <img src="img/about-office.png" alt="strategy">
+            <h2>Places We Work</h2>
+            <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+            <button>Learn More</button>
+          </div>
+          <div class="mR">
+            <img src="img/about-meeting.png" alt="strategy">
+            <h2>Collaboration</h2>
+            <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+            <button>Learn More</button>
+          </div>
+        </section>
+
+        <section class="bottom">
+          <h2>Let's Work Together</h2>
+          <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
+        </section>
+
+      </main>   
       <footer>
         <nav>
           <a href="index.html">Home</a>
-          <a href="#">About</a>
+          <a href="about.html">About</a>
```

Comment:
Good job remembering to add the link in the footer. A lot of people overlook that!